types/upgrade_risks_predictions.go

```diff
@@ -36,6 +36,13 @@ type UpgradeRisksPredictors struct {
 	OperatorConditions []OperatorCondition `json:"operator_conditions"`
 }
 
+// DataEngResponse is the response received from the data-eng service
```

Comment:
this should probably go into `types` repository

Reply:
it's used just in this repo. Is it needed to be moved there?